lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -104,7 +105,7 @@ def write_playbook
       end
 
       def start_ansible_runner
-        command = ['ansible-runner', 'run', @root, '-p', 'playbook.yml']
+        command = check_mode? ? ['ansible-runner', '--cmdline', '"--check"', 'run', @root, '-p', 'playbook.yml'] : ['ansible-runner', 'run', @root, '-p', 'playbook.yml']
```

Comment:
Could we keep the original command and if check mode is enabled, just append `--cmdline` and `"--check"` to it instead of having arrays which are mostly the same?

Reply:
Yes, I agree I will fix this.